src/OpenTelemetry/Logs/LogRecord.cs

```diff
@@ -247,6 +262,30 @@ public EventId EventId
             set => this.ILoggerData.Exception = value;
         }
 
+        /// <summary>
+        /// Gets or sets the original string representation of the severity as it is
+        /// known at the source.
+        /// </summary>
+        internal string? SeverityText
+        {
+            get => this.Data.SeverityText;
+            set => this.Data.SeverityText = value;
+        }
+
+        /// <summary>
+        /// Gets or sets the log <see cref="LogRecordSeverity"/>.
+        /// </summary>
+        internal LogRecordSeverity? Severity
```

Comment:
Nullable or just Unspecified?

Reply:
I feel like `Nullable` is more the .NET way for saying a value type is unspecified but originally I didn't have `LogRecordSeverity.Unspecified` defined 🤣 If you think it would be better to just default to that and drop Nullable I'm good with it.

Reply:
I'm ok leaving this for now. Good question though.. I think we should opt for one or the other: nullable or `Unspecified`. I think I'll open an issue where we can capture a punch list of small questions like this for us to circle back to before we declare stable.